aries_vcx/src/handlers/mod.rs

```diff
@@ -55,9 +55,8 @@ impl From<IssuerState> for u32 {
             IssuerState::Initial => 0,
             IssuerState::ProposalReceived => 1,
             IssuerState::OfferSet => 2,
-            IssuerState::OfferSent => 3,
             IssuerState::RequestReceived => 4,
-            IssuerState::CredentialSent => 5,
+            IssuerState::CredentialSet => 5,
```

Comment:
Probably just to state the obvious - all credentials in `OfferSent` and `CredentialSent` states will become invalid and will have to be discarded before updating to this version (and all code relying on the old states updated).

Reply:
Yes indeed. We could potentially keep CredentialSent (just keep it, but do not transition into it anymore through out the protocol) temporarily ease the transition and remove the state later.

Reply:
Let's just keep this in mind and make the next release a "major" one...